core/src/main/java/org/springframework/security/oauth2/server/authorization/web/OAuth2TokenEndpointFilter.java

```diff
@@ -29,16 +44,56 @@
 
 /**
  * @author Joe Grandja
+ * @author Madhu Bhat
  */
 public class OAuth2TokenEndpointFilter extends OncePerRequestFilter {
-	private Converter<HttpServletRequest, Authentication> authorizationGrantConverter;
+	private final String DEFAULT_TOKEN_URI = "/oauth2/token";
```

Comment:
Rename to `DEFAULT_TOKEN_ENDPOINT_URI` and make `static`. We should also provide a constructor that accepts `tokenEndpointUri` allowing for customization. See `OAuth2AuthorizationRequestRedirectFilter` in `spring-security-oauth2-client` for an example.